examples/xmp.rs

```diff
@@ -0,0 +1,1012 @@
+//! Analogue of fusexmp
+//!
+//! See also a more high-level example: https://github.com/wfraser/fuse-mt/tree/master/example
```

Comment:
I actually cribbed this right from the fuse-rs pr https://github.com/zargony/fuse-rs/pull/141/files, but its quite possibly out of date/incorrect, i'lll take a look tomorrow

Reply:
The link is intentional i think since this seems to be a port of the code in https://github.com/wfraser/fuse-mt/blob/master/example/src/passthrough.rs which is aimed at the libfuse C bindings. (I think the reference here is fusexmp is low level/C, the rust one is higher level?). Open to new wording or changing anything here.

Reply:
I'm actually in the process of changing `fuse-mt` over to `fuser` now fyi, heh

Reply:
@wfraser cool! btw, what do you think of making the `fuser` API `async`?